fluentd/templates/daemonset-fluentd.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2017 The Openstack-Helm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: fluentd
+  labels:
+    app: fluentd
+spec:
+  template:
+    metadata:
+      labels:
+        app: fluentd
+      annotations:
+        configmap-etc-hash: {{ tuple "configmap-etc.yaml" . | include "helm-toolkit.hash" }}
+    spec:
+      containers:
+      - name: fluentd
+        image: {{ .Values.images.fluentd }}
+        imagePullPolicy: {{ .Values.images.pull_policy }}
+        {{ if .Values.resources.enabled }}
+        resources:
+          limits:
+            cpu: {{ .Values.resources.limits.cpu }}
+            memory: {{ .Values.resources.limits.memory }}
+          requests:
+            cpu: {{ .Values.resources.requests.cpu }}
+            memory: {{ .Values.resources.requests.memory }}
+        {{ end }}
+        command:
+          - bash
+          - /tmp/start.sh
+        volumeMounts:
+        - name: varlog
+          mountPath: /var/log
+        - name: varlibdockercontainers
+          mountPath: /var/lib/docker/containers
+          readOnly: true
+        - name: etctdagent
+          mountPath: /etc/td-agent
+        - name: tdagentconf
+          mountPath: /etc/td-agent/td-agent.conf
+          subPath: td-agent.conf
+        - name: startsh
+          mountPath: /tmp/start.sh
+          subPath: start.sh
+          readOnly: true
```

Comment:
They're not appropriate here.  The only configuration file that needs to be mounted is td-agent.conf, and the only behavior we should support is the ability to override that file wholesale.